Api/Data/UpdateCartResultInterface.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2018 Bolt Financial, Inc (https://www.bolt.com)
```

Comment:
Could you please change all copyright to 2017-2020 as the PR https://github.com/BoltApp/bolt-magento2/pull/817 that @mccottry made

Reply:
Fixed